src/main/resources/attackvectors/UnionBasedSQLInjectionVulnerabilityPayload.properties

```diff
@@ -0,0 +1,7 @@
+UNION_BASED_SQL_INJECTION_PAYLOAD_LEVEL_1=By sending a query with <i>UNION</i> word we can gain data about users or other internal information from the databases. An example for this is:<br/>\
+<code>5 UNION SELECT * FROM users WHERE id=1</code><br/>\
+If you insert this in the textbox, you will get information about a user from the table.
```

Comment:
Add `similarly hacker can retrieve the entire data from the database which includes Credit card details, personally identifiable information etc of other users using the application`